### PR TITLE
Bridge between myjob 1.0 and 2.0

### PR DESCRIPTION
**Pull request by zifeo**

### CODE REVIEW

File: app/Http/Controllers/AdController.php [hunk 1]
```diff
@@ -11,6 +11,9 @@
 use Redirect;
 use Session;
 use Validator;
+use Request;
+use Mail;
```
Comment (timozattol) [inline app/Http/Controllers/AdController.php]: Do you use Request & Mail in the code?

File: app/Http/Controllers/AdController.php [hunk 2]
```diff
@@ -42,6 +45,42 @@ public function created() {
...
+			'salary' => $getOrElse('renumeration'),
+			'skills' => $getOrElse('competence'),
+			'languages' => $getOrElse('langues') == '2' ? 'français' : $getOrElse('langues') == '18' ? 'allemand' : 'anglais',
+			'contact_first_name' => $getOrElse('contact'),
+			'contact_last_name' => '',
+			'contact_email' => $email,
+			'contact_phone' => null
+		];
+
+		Log::info(print_r($data, true) . print_r($bridgedAd, true));
```
Comment (timozattol) [inline app/Http/Controllers/AdController.php]: maybe this Log could be "debug" and the next one could stay as "info".

File: app/Http/Controllers/AdController.php (hunk 1 above)
Comment (zifeo) [inline app/Http/Controllers/AdController.php]: Yes :)!

**Comment by timozattol** [comment]
I tried to pull the last version of the code on the vm, there was a merge conflict even though no commit was different. I don't understand what happened but I fixed it :grin:

**Comment by zifeo** [comment]
:+1:

**Pull request merged**